mikeio/__init__.py

```diff
@@ -150,25 +150,23 @@ def open(filename: str, **kwargs):
 
     >>> dfs = mikeio.open("pt_spectra.dfs2", type="spectral")
     """
-    ext = os.path.splitext(filename)[1].lower()
-
-    if ext == ".dfs0":
-        return Dfs0(filename, **kwargs)
-
-    elif ext == ".dfs1":
-        return Dfs1(filename, **kwargs)
-
-    elif ext == ".dfs2":
-        return Dfs2(filename, **kwargs)
-
-    elif ext == ".dfs3":
-        return Dfs3(filename, **kwargs)
-
-    elif ext == ".dfsu":
-        return Dfsu(filename, **kwargs)
-
-    elif ext == ".mesh":
-        return Mesh(filename, **kwargs)
-
-    else:
-        raise Exception(f"{ext} is an unsupported extension")
+    file_format = os.path.splitext(filename)[1].lower()[1:]
+
+    READERS = {
+        "dfs0": Dfs0,
+        "dfs1": Dfs1,
+        "dfs2": Dfs2,
+        "dfs3": Dfs3,
+        "dfsu": Dfsu,
+        "mesh": Mesh,
+    }
+
+    if file_format not in READERS:
+        valid_formats = ", ".join(READERS.keys())
+        raise Exception(
+            f"{file_format} is not a supported format for mikeio.open. Valid formats are {valid_formats}"
```

Comment:
Maybe add hint about mikeio.read_pfs for MIKE configuration files in the PFS format?